Tools/WinMLRunner/src/main.cpp

```diff
@@ -36,7 +50,9 @@ int main(int argc, char *argv[])
         wprintf(error.message().c_str());
         return error.code();
     }
-    int returnCode = run(*commandLineArgs, profiler, deviceList);
+    LearningModelSessionOptions sessionOptions;
+    PopulateSessionOptions(sessionOptions);
+    int returnCode = run(*commandLineArgs, profiler, deviceList, sessionOptions);
     free(commandLineArgs);
```

Comment:
Not related to your PR, but just saw this big red flag :( I can fix this in another PR